src/User/LoginForm.jsx

```diff
@@ -14,7 +16,12 @@ export const StyledLink = styled(Link)`
   padding: 10px;
 `;
 
-const LoginForm = ({ setSessionToken }) => {
+const LoginForm = () => {
+  const [, setCookie] = useCookies([cookieName]);
```

Comment:
missing the first param here

Reply:
Leaving out the first param here and only having a comma just means no warnings are thrown on the front end, but it works the same way with [cookies, setCookie, removeCookie]